Brokerages/InteractiveBrokers/InteractiveBrokersBrokerage.cs

```diff
@@ -907,6 +909,8 @@ public override void Dispose()
         {
             Log.Trace("InteractiveBrokersBrokerage.Dispose(): Disposing of IB resources.");
 
+            _isDisposeCalled = true;
```

Comment:
Minor but we might wanna make this variable `volatile` since both code paths will be run by different threads? Also since we are at it, we could add a check at the top it's already true just return